packages/perseus-core/src/parse-perseus-json/general-purpose-parsers/is-object.test.ts

```diff
@@ -0,0 +1,118 @@
+import * as fs from "fs";
+import {tmpdir} from "os";
+import {join} from "path";
+import {Worker} from "worker_threads";
+
+import {isObject} from "./is-object";
+
+describe("isObject", () => {
+    it("is true for a POJO", () => {
+        expect(isObject({})).toBe(true);
+    });
+
+    it("is false for an array", () => {
+        expect(isObject([])).toBe(false);
+    });
+
+    it("is false for a class instance", () => {
+        // Even if our class is named Object, it's not the same as the global
+        // Object.
+        class Object {}
+        expect(isObject(new Object())).toBe(false);
+    });
```

Comment:
I think the naming of `isObject` is a little confusing although I can't think of anything better off the top of my head. An array **_is_** an object, right? An instance of a class _**is**_ an object? Which is why I imagine we're not able to use `_.isObject` here (besides also trying to use less underscore).

`isPlainObject`? `isSerializableObject`? idk